easybuild/easyblocks/b/boost.py

```diff
@@ -158,7 +159,9 @@ def configure_step(self):
     def build_step(self):
         """Build Boost with bjam tool."""
 
-        bjamoptions = " --prefix=%s" % self.objdir
+        cxxflags = os.environ['CXXFLAGS']
+        ldflags = os.environ['LDFLAGS']
```

Comment:
@dolfim looks good, but it's safer to use `os.getenv` or `os.environ.get`, so you can specify a fallback value in case `$CXXFLAGS` or `$LDFLAGS` are not defined?

Or, alternatively, these should only be passed down *if* they are defined (which is usually the case in EasyBuild, but not always, e.g. when installing with `dummy` toolchain).